build.gradle

```diff
@@ -2,23 +2,15 @@
 
 buildscript {
```

Comment:
We need to remove ":app" from the settings.gradle file.  It's producing this warning:  Warning:<i><b>root project 'eosio-java-android-rpc-provider': Unable to resolve additional project configuration.</b>
Details: org.apache.tools.ant.BuildException: Basedir /Users/ken.scarberry/Documents/AndroidStudioProjects/eosio-java-android-rpc-provider/app does not exist</i>

Reply:
Should just need:  include ':eosiojavarpcprovider'